Tensile/GenerateSummations.py

```diff
@@ -47,6 +47,37 @@ def isValidArch(archName, currentArch):
     arch = gfxArch(archName)
     return currentArch == arch
 
+# Including readSolutionRaw in this file. It is not clear that
+# this functionality is needed outside the scope of this utility.
+# For this utility we need the unaltered data. 
+def readSolutionRaw(filename):
+    try:
+        stream = open(filename, "r")
+    except IOError:
+        print ("Cannot open file: %s" % filename )
+        return None
+    data = yaml.load(stream, yaml.SafeLoader)
```

Comment:
If you cannot read the file, ok to continue?

Reply:
Fixed below.